src/ontology/mondo-edit.obo

```diff
@@ -552390,7 +552390,8 @@ subset: rare
 xref: DOID:0081277 {source="MONDO:equivalentTo"}
 xref: MEDGEN:1808288 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: UMLS:C5669918 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1808288"}
-is_a: MONDO:0100342 {source="DOID:0081277"} ! malignant glioma
+is_a: MONDO:1010030 {source="DOID:0081277"} ! pediatric high-grade glioma
```

Comment:
DO does not say that this term is a child of pediatric high-grade glioma. 
Please keep the original SubClass of as it is the actual SC according to DO (it is redundant, but we like redundancy in Mondo :-) it also helps identify potential errors). 

Fro this new SC (pediatric high-grad glioma), please add the orcid

Reply:
Thanks for catching! Added back the original SC according to DO -> malignant glioma. 

For the new SC pediatric high-grade glioma, I added the orcid as the source.